eng/Versions.props

```diff
@@ -27,6 +27,10 @@
   <PropertyGroup>
     <SystemCollectionsImmutableVersion>6.0.0</SystemCollectionsImmutableVersion>
     <SystemConfigurationConfigurationManagerVersion>6.0.0</SystemConfigurationConfigurationManagerVersion>
+    <!-- 
+        Modifying the version of System.Memory is very high impact and often causes downstream breaks on repos that depend on us.
+        When updating the version of System.Memory, file a breaking change here: https://github.com/dotnet/docs/issues/new?assignees=gewarren&labels=breaking-change%2CPri1%2Cdoc-idea&template=breaking-change.yml&title=%5BBreaking+change%5D%3A+ 
```

Comment:
Is this sufficient? Should we also post a VS breaking change? Should we say "try not to do this but"?

Reply:
Not a bad idea. I searched around a bit, but couldn't find a link to file VS breaking changes. Do you have one handy?

Reply:
I think https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/1796/How-to-add-a-Known-Issue (internal to Microsoft) is the best we have